server/app/services/program/ProgramService.java

```diff
@@ -69,6 +70,8 @@ public final class ProgramService {
   private static final String MISSING_ADMIN_NAME_MSG = "A program URL is required";
   private static final String INVALID_ADMIN_NAME_MSG =
       "A program URL may only contain lowercase letters, numbers, and dashes";
+  private static final String INVALID_PROGRAM_SLUG_MSG =
+      "A program URL must contain at least one letter";
```

Comment:
Optional: What does the form text use? Does it call the field a URL?

Would something like "A program identifier must contain..." better align with the form?

Reply:
This error message is referring to the same input box as the ones on line 72 `INVALID ADMIN_NAME_MSG` and `MISSING_ADMIN_NAME_MSG` so I chose the same subject in the error message to align with them. 

Can change it if you feel strongly about changing the subject tho!

Reply:
Ah makes sense, that works for me.